app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    data = command.split()
+    file_to_copy = data[1]
+    copy = data[2]
+
+    if len(data) != 3 or data[0] != "cp" or file_to_copy == copy:
+        return
+
+    with open(file_to_copy, "r") as file_in, open(copy, "a") as file_out:
```

Comment:
The file is opened in append mode ('a'), which will add the content to the end of the file if it already exists. If the intention is to overwrite the file, consider using write mode ('w') instead.